website/docs/r/spacecraft.html.markdown

```diff
@@ -0,0 +1,93 @@
+---
+subcategory: "Orbital"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_spacecraft"
+description: |-
+  Manages a Spacecraft resource.
+---
+
+# azurerm_spacecraft
+
+Manages a Spacecraft.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "rg-example"
+  location = "West Europe"
+}
+
+resource "azurerm_spacecraft" "example" {
+  name                = "example-spacecraft"
+  resource_group_name = azurerm_resource_group.test.name
+  location            = "westeurope"
+  norad_id            = "12345"
+  links {
+    bandwidth_mhz        = 100
+    center_frequency_mhz = 101
+    direction            = "Uplink"
+    polarization         = "LHCP"
+    name                 = "examplename"
+  }
+  tle_line_1 = "1 23455U 94089A   97320.90946019  .00000140  00000-0  10191-3 0  2621"
+  tle_line_2 = "2 23455  99.0090 272.6745 0008546 223.1686 136.8816 14.11711747148495"
+  title_line = "AQUA"
+  tags = {
+    aks-managed-cluster-name = "9a57225d-a405-4d40-aa46-f13d2342abef"
+  }
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the Spacecraft. Changing this forces a new resource to be created.
+
+* `resource_group_name` - (Required) The name of the Resource Group where the Spacecraft exists.
+
+* `location` - (Required) The location where the Spacecraft exists.
+
+* `norad_id` - (Required) NORAD ID of the Spacecraft.
+
+* `links` - (Required) A `links` block as defined below.
+
+---
+
+* `bandwidth_mhz` - (Required) Bandwidth in Mhz.
+
+* `center_frequency_mhz` - (Required) Center frequency in Mhz.
+
+* `direction` - (Required) Direction if the communication. Possible values are `Uplink` and `Downlink`.
+
+* `polarization` - (Required) Polarization. Possible values are `RHCP`, `LHCP`, `linearVertical` and `linearHorizontal`.
+
+* `name` - (Required) Name of the link.
+
+---
+
+* `tle_line_1` - (Optional) The name of the field in output events used to specify the primary key which insert or update operations are based on.
```

Comment:
line 2 is missing here

how does TLE == `name of the field in output events used to specify the primary key which insert or update operations are based on.` could we expand on this and make it mroe clear how these are supposed to be set?

Reply:
Thanks for pointing out. This description is not accurate, will change.